javascript/javascript_in_the_real_world/linting.md

```diff
@@ -14,7 +14,7 @@ Code style is important! Having a consistent set of style rules for things such
 
 The style guides we mentioned above are full of really helpful advice for formatting, organizing and composing your code. But there are a _lot_ of rules - it can be difficult to internalize them all. **Linters** are tools that will scan your code with a set of style rules and will report any errors to you that they find. In some cases, they can even auto-fix the errors! The following articles explain in more detail the benefits of using a linter while you code.
 
-1. [This article](http://mikecavaliere.com/javascript-linting-what-developers-need-to-know/) gets right to the point... start here!
+1. [This article](https://web.archive.org/web/20220113100650/https://mikecavaliere.com/javascript-linting-what-developers-need-to-know/) gets right to the point... start here!
```

Comment:
I don't think we'll need to use the archive link, as I was able to find the article on the same website. Looks like they moved their articles to `/posts`: https://mikecavaliere.com/posts/javascript-linting-what-developers-need-to-know

Reply:
Hey that's great! Didn't realise that was the case, thanks!

Reply:
Would you be willing to make that change?

Reply:
I've made the changes! Sorry it took me awhile to figure this out

Reply:
No worries, thanks for contributing :rocket:

Reply:
Hey, my apologies, I didn't realise that the links under Knowledge check linked to the same webpage and didn't update them. Should I open a new pull request?

Reply:
Thanks for checking. Yes, opening a new pull request will be ideal.

Reply:
Okay, I've opened a new pull request for this matter